bundles/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/SystemInfoThingTypeProvider.java

```diff
@@ -106,59 +105,34 @@ public boolean updateThingType(ThingTypeUID typeUID, List<ChannelGroupDefinition
                 return false;
```

Comment:
This should not happen, so you can just log the warn and return.

Reply:
Ok. I was able to remove more not used code

Reply:
Done.